--- /usr/src/linux/drivers/usb/storage/unusual_devs.h	2009-11-10 14:05:51.000000000 +0100
+++ /usr/src/linux/drivers/usb/storage/unusual_devs.h	2009-11-22 22:45:28.000000000 +0100
@@ -1337,6 +1337,13 @@
 		US_SC_DEVICE, US_PR_DEVICE, NULL,
 		US_FL_IGNORE_DEVICE),
 
+/* Reported by anomen */
+UNUSUAL_DEV(  0x0fce, 0xe0f3, 0x0000, 0x0000,
+		"Sony Ericsson",
+		"W595",
+		US_SC_DEVICE, US_PR_DEVICE, NULL,
+		US_FL_FIX_CAPACITY | US_FL_NO_WP_DETECT | IGNORE_RESIDUE ),
+
 /* Reported by Jan Mate <mate@fiit.stuba.sk>
  * and by Soeren Sonnenburg <kernel@nn7.de> */
 UNUSUAL_DEV(  0x0fce, 0xe030, 0x0000, 0x0000,
